src/components/Timeline/package.json

```diff
@@ -25,8 +25,7 @@
         "@gemeente-denhaag/baseprops": "^0.1.7",
         "@gemeente-denhaag/icons": "^0.1.0",
         "@gemeente-denhaag/typography": "^0.1.7",
-        "@material-ui/core": "^4.11.0",
-        "clsx": "^1.1.1"
+        "@material-ui/core": "^4.11.0"
     },
```

Comment:
if you are going to remove clsx in this PR, also remove any imports of clsx in the Timeline component

Reply:
I first installed it only for Timeline, but then realized I wanted to install it for every component. See commit 
d925c9f